src/Examples/KP2/Representation.h

```diff
@@ -0,0 +1,16 @@
+#ifndef KP2_REPRESENTATION_H_
+#define KP2_REPRESENTATION_H_
+
+#include <vector>
+#include <ostream>
+
+using RepKP2 = std::vector <bool>;
```

Comment:
Very good Anderson!! I'll start using this too... so much better than typedef ;)